saleor/order/utils.py

```diff
@@ -218,24 +216,19 @@ def update_taxes_for_order_line(
         else total_price
     )
     if unit_price.tax and unit_price.net:
-        line.tax_rate = manager.get_order_line_tax_rate(
-            order, product, variant, None, unit_price
-        )
+        line.tax_rate = calculations.order_line_tax_rate(order, line, manager)
 
 
-def update_taxes_for_order_lines(
+def update_order_lines_prices_if_expired(
     lines: Iterable[OrderLine], order: "Order", manager, tax_included
 ):
     for line in lines:
-        update_taxes_for_order_line(line, order, manager, tax_included=tax_included)
+        update_order_line_prices_if_expired(
+            line, order, manager, tax_included=tax_included
+        )
     OrderLine.objects.bulk_update(
         lines,
         [
-            "tax_rate",
-            "unit_price_net_amount",
-            "unit_price_gross_amount",
-            "total_price_net_amount",
-            "total_price_gross_amount",
             "undiscounted_unit_price_gross_amount",
```

Comment:
What is a benefit of having part update of prices fields in order.calculations file and the rest of them here? Wouldn't be good to extend functions in order.calculations and include them a rest of the methods responsible for returning prices? Fields like `undiscounted_unit_price...`.